popdat/scripts/build_population_areas.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -e
+
+# Outputs a FlatGeoBuf of geometries and their corresponding population,
+# suitable for use in the popdat crate.
```

Comment:
This script is kind of heinous. I struggled a bit with ogr2ogr, but I'm not sure how to make it better.

Part of this would be straight forward to port to rust, but unfortunately rust doesn't yet have a a FGB writer.